crates/oxc_transformer/src/lib.rs

```diff
@@ -44,6 +45,7 @@ pub use crate::{
 };
 
 pub struct Transformer<'a> {
+    ctx: Rc<RefCell<TransformerCtx<'a>>>,
```

Comment:
This doesn't look right, why is wrapped again when everything inside it has a `Rc` already?

Reply:
If we need to place the ctx into a struct we have to do this. Because we cannot move ctx to struct(We need to pass the ctx reference to each plugin) or put the reference of ctx to struct(local reference cannot do it).

Reply:
> We need to pass the ctx reference

Everything inside it has a `Rc` so it's free to `.clone()` and pass it around.

Reply:
> Everything inside it has a `Rc` so it's free to `.clone()` and pass it around.

Ah... Yes. Thanks for the reminder. 

After all my code changes, I forgot that I could use `.clone`. It took me a long time to come up with this solution to the problem...